Services/BackgroundTasks/classes/class.ilBTPopOverGUI.php

```diff
@@ -76,13 +76,20 @@ public function getItemForObserver(Bucket $observer) : ILIAS\UI\Component\Item\N
         $icon = $f->symbol()->icon()->standard("bgtk", $this->txt("bg_task"));
         $title = $observer->getTitle() . ($state === State::SCHEDULED ? " ({$this->txt('scheduled')})" : "");
 
+        $item = $f->item()->notification($title, $icon);
+
+        $description_text = $this->txt('background_tasks');
+        $item = $item->withAdditionalOnLoadCode(function ($id) use ($description_text) {
+            return "il.BGTask.updateDescriptionOnClose('#$id','$description_text');";
+        });
+
         if ($state === State::USER_INTERACTION) {
             $actions = $this->getUserInteractionContent($observer, $redirect_uri);
             $primary_action = array_pop($actions);
             if ($primary_action instanceof Button) {
                 $title = $primary_action->withLabel($title);
             }
-            $item = $f->item()->notification($title, $icon);
```

Comment:
We can't remove this because $title might be adjusted in this scenario. It'd probably be best to not return the $item within this if-branch and add the closure that updates the description on close right before the last return statement.

Reply:
Good catch.